Dockerfile.openpilot_base

```diff
@@ -1,3 +1,7 @@
+FROM condaforge/mambaforge as pocl-builder
+
+RUN mamba install pocl
```

Comment:
this conda POCL package has opencl drivers for nvidia cuda, intel, and amd

Reply:
how big is it?